generators/generator-microsoft-bot-conversational-core/generators/app/index.js

```diff
@@ -2,44 +2,50 @@
 // Licensed under the MIT License.
 
 'use strict';
-const Generator = require('yeoman-generator');
 
-module.exports = class extends Generator {
-  constructor(args, opts) {
-    super(args, opts);
+const {
+  BaseGenerator,
+  platforms,
+} = require('@microsoft/generator-microsoft-bot-adaptive');
 
-    this.argument('botName', { type: String, required: true });
-  }
+const packageReferences = {
+  [platforms.dotnet]: {
+    version: '1.0.0-preview.20210310.8ee9434',
+    packages: [
+      'Microsoft.Bot.Components.HelpAndCancel',
+      'Microsoft.Bot.Components.Welcome',
+    ],
+  },
```

Comment:
We should probably handle this as an array on the basis that we may want to support different package versions. Alternatively, refactor back to have version explicitly defined for each.

Reply:
Sure, I'll revert to the old pattern.